packages/kit/src/views/Market/MarketDetailV2/components/TokenSecurityAlert/useTokenSecurity.ts

```
@@ -0,0 +1,217 @@
+  const isWarningValue = (value: string | undefined): boolean => {
+    return value === '1' || value === 'true';
+  };
+
+  const items: ISecurityKeyValue[] = [];
+
+  // Always show tax information
+  items.push({
+    key: 'buyTax',
+    label: 'Buy Tax',
+    value: formatPercentage(data.buyTax),
+    isWarning: parseFloat(data.buyTax || '0') > 10,
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)